create_framework/event_dispatcher.rst

```diff
@@ -45,6 +45,9 @@ the Response instance::
 
     class Framework
     {
+        
+    	/* all properties are now declared as private, they were protected until this chapter -> either there is a good reason and it might be useful to explain this change, or there could be better coherence by keeping them protected *or* private in the entire book */
+
```

Comment:
Nice catch, I added a PR https://github.com/symfony/symfony-docs/pull/14501 are there any other occurrences I didn't catch?

Reply:
Excellent. These lines can now be removed.

Well done @OskarStark and @Ca-Jou 👍

Reply:
@OskarStark not that I'm aware of.
@Nyholm sorry for the newb question, but it's my first PR ever: am I supposed to remove these lines in my PR (by committing once more or editing this commit) now that another one has been opened?

Reply:
> sorry for the newb question, but it's my first PR ever:

Congratulations. And what a great PR to start with =)

You can just make a new commit and push to the same branch (patch-create-framework). There is no need to be fancy and editing the commits etc.